src/Lib/LanguagesLib.php

```diff
@@ -617,7 +617,7 @@ public static function languagesInTatoeba()
                 'nus' => __d('languages', 'Nuer'),
                 'ckb' => __d('languages', 'Central Kurdish (Soranî)'),
                 'kmr' => __d('languages', 'Northern Kurdish (Kurmancî)'),
-                'sdh' => __d('languages', 'Southern Kurdish (Xwarîn)'),
+                'sdh' => __d('languages', 'Xwarîn'),
```

Comment:
This should be the other way around: we just want `Southern Kurdish` as the language name.

Reply:
Oh, thanks!